### ISSUE TITLE
teb1 recognition motif

### ISSUE THREAD

**Issue by Antonialock** [The-Sequence-Ontology/SO-Ontologies#439]
Hi could I please request Teb1 recognition motif
motif: TTAGGG
PMID:23314747

thanks!

**Comment by davidwsant** [comment]
Hi Antonia,

I apologize that this has taken so long for us to respond. We had a change in staff and now I am working on old requests that never received responses. 

Based on what you have posted above, I have come up with this:

name: teb1_recognition_motif
def: "A binding motif with the consensus sequence TTAGGG to which Teb1 binds." PMID:23314747
synonym: "teb1 recognition motif" 
is_a: SO:0000713 ! DNA_motif

Does this all look good to you? Do you think any more about be added in the definition about teb1 and how it controls histone binding? 

Thanks,

Dave

**Comment by Antonialock** [comment]
Is this model useful?
<img width="752" alt="Screenshot 2020-04-26 at 07 06 46" src="https://user-images.githubusercontent.com/12727506/80299472-884adc00-878c-11ea-99d9-3c1053a97131.png">

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5128866/

I don't know how conserved the exact mechanism is.

**Comment by davidwsant** [comment]
Hi Antonia,

I apologize that I did not add this before I added your more recent requests. After replying to several of the issue requests I started working again from newest terms backward. The term teb1_recognition_motif has been added with the ID SO:0002258. The new term should be visible in the [SO Browser](http://www.sequenceontology.org/browser/current_svn/) within about 24 hours.

Thank you,

Dave
